Solutions/CohesitySecurity/Playbooks/SNOW-CreateAndUpdateIncident/readme.md

```diff
@@ -0,0 +1,24 @@
+# Create Incident in Service Now Playbook
+## Summary
+This playbook creates a ticket in ServiceNow
+
+## Prerequisites
+1. Cohesty SIEM/SOAR integration needs to be deployed prior to the deployment of this playbook under the same subscription.
```

Comment:
@yinghuang123, please fix the typo Cohesty to Cohesity, thanks.

Reply:
The file was removed but the content was fixed in the new place.